sql/catalyst/src/main/scala/org/apache/spark/sql/types/numerics.scala

```diff
@@ -22,7 +22,7 @@ import scala.math.Ordering
 
 import org.apache.spark.sql.types.Decimal.DecimalIsConflicted
 
-object ByteExactNumeric extends ByteIsIntegral with Ordering.ByteOrdering {
+private[sql] object ByteExactNumeric extends ByteIsIntegral with Ordering.ByteOrdering {
```

Comment:
I'm OK with it. But we should also move this file to a private package.

Reply:
we can move it in followup

Reply:
Got it, others object in numerics.scala might also get moved together.